src/components/Snippet/FileManager.vue

```diff
@@ -150,6 +161,22 @@ export default {
         this.postUpsert(),
       ]);
     },
+    rowRightClick(row, e) {
+      // currently there is only one item on the menu. so when no access control, there is no menu.
+      if (this.noAccessControl) {
+        return;
+      }
+      if (['collection', 'folder'].indexOf(row._modelType) !== -1) {
+        e.preventDefault();
+        this.collectionAndFolderMenu.show = false;
+        this.collectionAndFolderMenu.x = e.clientX;
+        this.collectionAndFolderMenu.y = e.clientY;
+        this.actOnItem = row;
+        this.$nextTick(() => {
+          this.collectionAndFolderMenu.show = true;
+        });
+      }
+    },
```

Comment:
Seems like we ought to integrate this with the regular file browser better, but that shouldn't hold this up.

Reply:
I agree. This doesn't look so good.